compositor_api/src/input/whip.rs

```diff
@@ -18,6 +18,10 @@ pub struct WhipInput {
     /// Token used for authentication in WHIP protocol. If not provided, the random value
     /// will be generated and returned in the response.
     pub bearer_token: Option<Arc<str>>,
+    /// Internal use only.
+    /// Specifies prefix that is used in `input_id`.
```

Comment:
This api does not make sense, why are you defining an option that tells what prefix should be removed, if you can define an option that just says what id should be used